query-engine/query-engine/src/server/mod.rs

```diff
@@ -197,10 +197,20 @@ async fn request_handler(cx: Arc<PrismaContext>, req: Request<Body>) -> Result<R
 
                 Ok(res)
             }
-            Err(_e) => {
+            Err(e) => {
+                let ufe: user_facing_errors::Error = request_handlers::HandlerError::query_conversion(format!(
```

Comment:
Is this the only error that can happen in the code above?

Reply:
Yes. That's the catchall. The rest of the errors will be rendered as part of the usual request / response protocol, (i.e. a 200OK response with an `errors` key)